styles/style.css

```diff
@@ -0,0 +1,210 @@
+/* * {
+  border: black 0.5px solid;
+} */
+
+body {
+  font-family: 'Roboto', sans-serif;
+  background-color: seashell;
+}
+
+.menu {
+  display: grid;
+  grid-template-columns: 3fr 1fr 1fr;
+  margin: auto;
+  padding: 10px 10px 10px 10px;
+  list-style-type: none;
+  position: fixed;
+  top: 0;
+  width: 100%;
+
+}
+
+h1 {
+  font-size: 4em;
+  font-family: 'Muli', sans-serif;
+}
+
+.welcome {
+  display: flex;
+  flex-flow: column wrap;
+  align-content: space-between;
+  text-align: center;
+  font-family: Arial, bold, sans-serif;
+  margin: 180px 0px 0px 0px;
+  padding: 5px 0px 15em 0px;
+}
+
+.welcome sub {
+  font-size: 1.2em;
+  font-family: 'Noto Serif JP', sans-serif;
+  font-weight: bold;
+  text-transform: capitalize;
+  font-stretch: ultra-expanded;
+  margin-bottom: 2em;
+}
+
+.welcome a[href] {
+  font-size: .9rem;
+  color: gray;
+  margin: auto;
+  border: 2px darkgray solid;
+  border-radius: 10px;
+  padding: 5px;
+  text-decoration: none;
+  max-width: 10em;
+  background-color: white;
+}
+
+.quote {
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  align-content: space-evenly;
+  justify-content: center;
+  font-style: italic;
+  font-size: 20px;
+  font-family: 'Times New Roman', cursive;
+  margin: 0px 0px 100px 0px;
+  line-height: 2em;
+  padding: 6em 5px 5em 5px;
```

Comment:
hi, can you teach me how you solved your margin/padding issues so that the background color extended horizontally instead of lose space?

Reply:
It may have been exactly what you noted here. I made margins 0 (margin: top, right, bottom, left) and sometimes padding 0 all around too. I noticed, I had some issues with this on the personal site project and I found that it might have been from a sizing issue with some of box contents.